aopy/analysis/behavior.py

```diff
@@ -337,4 +337,48 @@ def get_cursor_leave_time(cursor_traj, samplerate, target_radius):
         leave_idx = np.where(dist>target_radius)[0][0]
         cursor_leave_time.append(t_axis[leave_idx])
     
-    return np.array(cursor_leave_time)
\ No newline at end of file
+    return np.array(cursor_leave_time)
+
+'''
+Continuous tracking behavioral metrics
+'''
+def calc_tracking_error(user_traj, target_traj):
+    '''
+    Computes the mean-squared error between the user position and target position over time.
+
+    Args:
+        user_traj (nt,ndim): user trajectory over a trial segment
+        target_traj (nt,ndim): target trajectory over a trial segment
+
+    Returns:
+        tracking_error (ndim,): array of floats corresponding to the tracking error in each dimension
+    '''
+    assert len(user_traj) == len(target_traj), "User and target trajectories must be the same length!"
+    return np.mean((user_traj - target_traj)**2, axis=0) # compute mean over time axis
+
+def calc_tracking_in_time(event_codes, event_times, proportion=False):
```

Comment:
why do you use the task events instead of the user and target trajectories? what if you want to only analyze a small chunk of time, for example a single trial, would this be possible?

Reply:
I think it's simpler to use events because there are events for specifically when the cursor enters and leaves the target. If I use trajectories, I need to account for the target and cursor radii. I wrote this function intending it to be use on a single trial basis, like in my test function!

Reply:
ok makes sense!